lighthouse-core/gather/gatherers/dobetterweb/document-event-listeners.js

```diff
@@ -115,17 +127,24 @@ class PageLevelEventListeners extends Gatherer {
   }
 
   afterPass(options) {
+    const driver = options.driver;
+
     return this.unlistenForScriptParsedEvents(options.driver)
-      .then(_ => this.collectListeners(LISTENER_LOCATIONS))
-      .then(listeners => {
-        this.artifact = listeners;
-      }).catch(_ => {
-        this.artifact = {
-          usage: -1,
-          debugString: 'Unable to gather passive events listeners usage.'
-        };
-      });
+        .then(_ => driver.sendCommand('DOM.enable'))
```

Comment:
what do you need the enable for? it doesnt look like you need events. 

if you do need this domain, you should probably disable it once you're done with it.